### ISSUE TITLE
[XMPP] no Autoreconnect

### ISSUE THREAD

**Issue by ghost** [loqui/im#445]
Sometimes the XMPP account switches from connected (green) to disconnected (yellow)

```
Firefox can't establish a connection to the server at wss://websockets.loqui.im/. strophe.js:4630
```

Autoreconnect is enabled under Settings but the account will not automatic reconnected.

If i close Loqui IM and start it again, the account switches to connected, again. I also could use the connect button to connect the account manually.

**Comment by ghost** [comment]
Same behaviour with loqui 0.3.1 on peak 1.1.1

**Comment by ghost** [comment]
Same issue with 0.3.2a. 

After few minutes running in the background, loqui loses connection and does not autom. reconnects. 

The "close" button does not react either, so the only remaining option is to restart again.

**Comment by cmeerw** [comment]
might be fixed with #772 and #776
